middle_end/flambda/simplify/env/downwards_env.ml

```diff
@@ -28,6 +28,15 @@ type resolver = Compilation_unit.t -> Flambda_type.Typing_env.t option
 type get_imported_names = unit -> Name.Set.t
 type get_imported_code = unit -> Exported_code.t
 
+type closure_info =
```

Comment:
Please put this in its own module `Closure_info`, so we can have e.g. `Closure_info.print`.

Reply:
done